frontend/src/actions/users.js

```diff
@@ -66,6 +67,33 @@ export function fetchMe() {
   };
 }
 
+//request to update email and nickname of user in account page
+export function updateMe() {
+  return (dispatch, getState) => {
+    let meState = getState().user.me;
+    console.log(meState);
```

Comment:
Lets not commit these log messages to master. If you want to include log messages in master, use a more substantial log message.